automation-roles/40-configure-infra/nfd-operator/templates/nfd-cr.j2

```diff
@@ -8,7 +8,7 @@ spec:
   instance: "" # instance is empty by default
   topologyupdater: false # False by default
   operand:
-{% if _p_current_ocp_version == '4.15' %}
+{% if _p_current_ocp_version|float() >= 4.15 %}
```

Comment:
There's no need to cast to float, you can just compare with > '4.15'; this is done by other parts of the code.

Reply:
Ah ok. I removed it but didn't test the change. If you think it's good then it can be merged.